client/src/styles/brand.scss

```diff
@@ -25,26 +25,21 @@
   &__logo + &__name {
     display: none;
   }
-
-  // Helper classes for Dark Mode support on image-based logos
-  &__logo--dark-mode {
-    display: none;
-  }
-
-  @media (prefers-color-scheme: dark) {
-    &__logo--light-mode {
-      display: none;
-    }
-
-    &__logo--dark-mode {
-      display: block;
-    }
-  }
 }
 
 .silverstripe-brand {
   color: var(--color-page-text);
   display: flex;
   justify-content: center;
   padding: var(--spacing-silverstripe-brand);
+
+  svg {
```

Comment:
I'd suggest adding a class to the `<svg>` element (something like `.silverstripe-cms-logo`) and specifying that here - otherwise you're targeting a generic element type and it runs a small but not insignificant risk of modifying any custom SVG that replaces the default. This is why the CSS lived directly in the SVG before - to isolate its specific colour requirements from the main spreadsheet.

(In retrospect, we should have named `.silverstripe-brand` something more like `.vendor-brand` to more accurately designate what belongs here - by default, the Silverstripe CMS logo, but optionally the logo of the vendor who built the site instead.)

Reply:
Are you happy if I put a class `.silverstripe-cms-logo` into the SilverStripeLogo.ss template to account for this?

Reply:
Yep, on the `<svg>` element :)

Reply:
Fixed

Reply:
@Cheddam / @ScopeyNZ are you still happy to merge this?

Reply:
Isn't this definition still missing `.silverstripe-cms-logo`.

Reply:
Sorry @elliot-sawyer, yes this should be changed to `.silverstripe-cms-logo` and the dark mode rule should drop the `svg` selector. Let me know as soon as you make that tweak and I'll merge :)